.env.example

```diff
@@ -43,3 +43,7 @@ CLOUDFLARE_ACCOUNT_TOKEN=v1.0-abcdef1234567890abcdef1234567890abcdef1234567890ab
 # The following variables specify instruction sets and configuration for the various models in SpongeChat.
 ## The value of the variable should correspond to the key of its' responding configuration in modelConstants.js.
 MODEL_LLM_PRESET=default
+
+# !! Pastebin
+
+WASTEBIN_HOST=https://localhost:8088
```

Comment:
This default needs to be changed to http. More documentation should also be added to address a docker-compose situation.